packages/g/gdu/package.yml

```diff
@@ -0,0 +1,24 @@
+name       : gdu
+version    : 5.31.0
+release    : 1
+source     :
+    - https://github.com/dundee/gdu/archive/refs/tags/v5.31.0/gdu-5.31.0.tar.gz : e3727680ea346ce86e63d4c97841cbc5e17c6d8e58fac8b8e9886e3339214e9d
+homepage   : https://github.com/dundee/gdu
+license    : MIT
+component  : system.utils
+summary    : Pretty fast disk usage analyzer written in Go.
+description: |
+    Gdu is intended primarily for SSD disks where it can fully utilize parallel processing...
+networking : true
+builddeps  :
+    - golang
+    - git
+build      : |
+    go build \
+        -buildmode=pie \
+        -ldflags="-s -w -linkmode external" \
+        -o gdu \
+        ./cmd/gdu
+install    : |
+    install -Dm00755 gdu $installdir/usr/bin/gdu
+    install -Dm644 gdu.1 ${installdir}/usr/share/man/man1/gdu.1
```

Comment:
The permissions passed to `install` should include all the numbers, in this case 00644.

`${installdir}` should also be just `$installdir`.